src/test/groovy/org/prebid/server/functional/tests/OrtbConverterSpec.groovy

```diff
@@ -1053,7 +1064,55 @@ class OrtbConverterSpec extends BaseSpec {
         then: "BidderRequest should contain the regs.gpp and regs.gppSid as on request"
         verifyAll(bidder.getBidderRequest(bidRequest.id)) {
             regs.gpp == bidRequest.regs.gpp
-            regs.gppSid.eachWithIndex { Integer value, int i -> bidRequest.regs.gppSid[i] == value}
+            regs.gppSid.eachWithIndex { Integer value, int i -> bidRequest.regs.gppSid[i] == value }
+        }
+    }
+
+    def "PBS should remove imp[0].refresh/qty when we don't support ortb 2.6"() {
+        given: "Default bid request with imp[0].refresh/qty"
+        def bidRequest = BidRequest.defaultBidRequest.tap {
+            imp[0].tap {
+                refresh = new Refresh(count: PBSUtils.randomNumber, refSettings: [new RefSettings(
+                        refType: PBSUtils.getRandomEnum(RefType),
+                        minInt: PBSUtils.randomNumber)])
+                qty = new Qty(multiplier: PBSUtils.randomDecimal,
+                        sourceType: PBSUtils.getRandomEnum(SourceType),
+                        vendor: PBSUtils.randomString)
+            }
+        }
+
+        when: "Requesting PBS auction with ortb 2.5"
+        prebidServerServiceWithElderOrtb.sendAuctionRequest(bidRequest)
+
+        then: "BidResponse shouldn't contain the imp[0].refresh/qty as on request"
+        def bidderRequest = bidder.getBidderRequest(bidRequest.id)
+        assert !bidderRequest.imp[0].refresh
+        assert !bidderRequest.imp[0].qty
```

Comment:
I suggest using `verifyAll` instead of multiple `assert` statements to be consistent with the existing code in this Spec